harvester/create.go

```diff
@@ -27,27 +27,31 @@ func (d *Driver) PreCreateCheck() error {
 		return err
 	}
 
+	// server version
+	serverVersion, err := c.Settings.Get("server-version")
+	if err != nil {
+		return err
+	}
+	d.ServerVersion = serverVersion.Value
+	if strings.HasPrefix(d.ServerVersion, "v0.1.0") {
+		return fmt.Errorf("current harvester server version is %s, only support v0.2.0+", d.ServerVersion)
+	}
```

Comment:
so we are deprecated v0.1.0 support of the harvester node driver?

Reply:
yes, because the vmimage crd has changed.